systemd/yamuisplash.service

```diff
@@ -1,14 +1,16 @@
 [Unit]
 Description=Simple yamui splash screen for SailfishOS
 Before=droid-hal-init.service
+Before=sailfish-unlock-agent.service
+Before=start-user-session.service
```

Comment:
I have a feeling that something should be done about that "Before=droid-hal-init.service".

The change in semantics of when splashscreen is "started" makes sense in context of ui chaining (splashscreen needs to reach "can be replaced" state before allowing starting of things that will replace it). But in case of droid hal, we just want to be grab display before droid things...

Reply:
NB there was explicit 5 second delay in ExecStartPost that got eliminated, so in practice the timings do not really change.

Reply:
Filed separate task about investigating whether the droid hal init delay needs to be as long as it is now.